tests/cli/test_local.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from contextlib import contextmanager
+from subprocess import CompletedProcess, PIPE
+from milatools.cli.local import CommandNotFoundError, Local
+
+
+_ECHO_CMD=["echo", "--arg1", "val1", "--arg2=val2", "X"]
+_FAKE_CMD=["FAKEcmd", "--arg1", "val1", "--arg2=val2", "X"]
+_FAIL_CODE_CMD=["FAKEcode", "--arg1", "val1", "--arg2=val2", "X"]
+
+
+def _test_stdouterr(func, capsys, file_regression):
+    out, err = None, None
+    try:
+        out, err = func()
+        if isinstance(out, CompletedProcess):
+            out, err = out.stdout, out.stderr
+    finally:
+        captured = capsys.readouterr()
```

Comment:
This will not capture ANSI escape sequence unless `pytest` is executed with the flag `capture=no`

Reply:
@breuleux , forgot to ask if you wanted to change how `pytest` is executed to print ANSI characters as well? This would allow to test how stuff are printed as well as what is printed but would make the output less readable